google/resource_google_folder_test.go

```diff
@@ -0,0 +1,163 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"os"
+	"testing"
+
+	resourceManagerV2Beta1 "google.golang.org/api/cloudresourcemanager/v2beta1"
+)
+
+func TestAccGoogleFolder_rename(t *testing.T) {
+	skipIfEnvNotSet(t, "GOOGLE_ORG")
+
+	folderDisplayName := "tf-test-" + acctest.RandString(10)
+	newFolderDisplayName := "tf-test-renamed-" + acctest.RandString(10)
+	org := os.Getenv("GOOGLE_ORG")
+	parent := "organizations/" + org
+	folder := &resourceManagerV2Beta1.Folder{}
```

Comment:
Another nit; I've seen other tests declare the resource variable as the struct type, not the pointer type (e.g. `resourceManagerV2Beta1.Folder` rather than `*resourceManagerV2Beta1.Folder`. The pointer allocation here feels a bit less explicit. But if this was intentional go ahead and leave it in as this is mainly stylistic.

Reply:
Done